v_htmlescape/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "v_htmlescape"
-version = "0.3.2"
+version = "0.5.0"
```

Comment:
do the versions really need to track the v_escape one? they should be on their own release schedule I think

Reply:
Seem right. Thank you.